.github/workflows/update-contributors.yml

```diff
@@ -31,6 +31,32 @@ jobs:
           CONTRIBUTORS_SECTION_PATTERN: "## GitHub Action Contributors"
           INPUT_TEMPLATE_FILE: ./templates/contributor-template.md
           FILE_WITH_PATTERN: ./index.md
+      - name: Get Hyperledger contributors information
+        uses: "hyperledger-tooling/github-contributors-action@main"
+        env:
+          GITHUB_AUTH_TOKEN: "${{ secrets.PAT }}"
+          SOURCE_GITHUB_REPOSITORY: hyperledger
+          CONTRIBUTORS_SECTION_PATTERN: "## Hyperledger GitHub Contributors"
+          CONTRIBUTORS_SECTION_END_PATTERN: "## Hyperledger Labs GitHub Contributors"
+          INPUT_TEMPLATE_FILE: ./templates/contributor-template.md
+          FILE_WITH_PATTERN: ./community.md
+      - name: Get Hyperledger Labs contributors information
+        uses: "hyperledger-tooling/github-contributors-action@main"
+        env:
+          GITHUB_AUTH_TOKEN: "${{ secrets.PAT }}"
+          SOURCE_GITHUB_REPOSITORY: hyperledger-labs
+          CONTRIBUTORS_SECTION_PATTERN: "## Hyperledger Labs GitHub Contributors"
+          CONTRIBUTORS_SECTION_END_PATTERN: "## Hyperledger Tooling GitHub Contributors"
+          INPUT_TEMPLATE_FILE: ./templates/contributor-template.md
+          FILE_WITH_PATTERN: ./community.md
+      - name: Get Hyperledger Tooling contributors information
+        uses: "hyperledger-tooling/github-contributors-action@main"
+        env:
+          GITHUB_AUTH_TOKEN: "${{ secrets.PAT }}"
+          SOURCE_GITHUB_REPOSITORY: hyperledger-tooling
+          CONTRIBUTORS_SECTION_PATTERN: "## Hyperledger Tooling GitHub Contributors"
+          INPUT_TEMPLATE_FILE: ./templates/contributor-template.md
+          FILE_WITH_PATTERN: ./community.md
```

Comment:
So we want to keep the contribution info on the tooling section in the index page as well as the community.md page?

Reply:
We can remove `index.md` in future PRs. Let's create a feature request to have top happenings/trends within Hyperledger community shown here.